cli.ts

```diff
@@ -413,6 +413,7 @@ const addTask = (cliConfig: CLIConfig, config: ConfigArgs, env: EnvVars, parsedA
             )
         },
         handler: (inputArgs: Record<string, unknown>) => {
+            // console.log("======JCC\n", inputArgs.storage, "\n======")
```

Comment:
I think this can be removed if it is no longer being used

Reply:
Will reflect in the next commit